A4_3.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+int+push;
+
+public class A4_3 {
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		Scanner scan = new Scanner(System.in);
+		System.out.print("Enter a number greater than 0 (less than 1 to quit): ");
+		
+		int num = scan.nextInt();
+		int sum = 0;
+		int square;
+		
+		if(num > 1 || num == 1) {
+			for (int n =1 ; n < num+1 ; n++ ) {
+				square = n * n;
+				sum += square;
+			}
+			System.out.printf("The sum of the squares from 1 to %d",num);
+			System.out.printf(" is %d",sum);
+		
+		}
+		
+		
+			
+	}
+
+}
+
+
+/* Enter a number greater than 0 (less than 1 to quit): 4

```

Comment:
did you forget to remove?

Reply:
the teacher is always asking for screen output noted as a comment at the bottom of the code. that's why.